jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsToolkitExplorerFactory.kt

```diff
@@ -53,8 +52,14 @@ class AwsToolkitExplorerFactory : ToolWindowFactory, DumbAware {
             )
         }
 
-        // content will be added to tool window in initialization of AwsToolkitExplorerToolWindow
-        AwsToolkitExplorerToolWindow.getInstance(project)
+        val contentManager = toolWindow.contentManager
+        val content = contentManager.factory.createContent(AwsToolkitExplorerToolWindow.getInstance(project), null, false).also {
+            it.isCloseable = true
+            it.isPinnable = true
+        }
+        contentManager.addContent(content)
```

Comment:
Should the next set of commands be run explicitly in edt or does this function already run on the ui thread?

Reply:
yeah `createToolWindowContent` is already on EDT